### PR TITLE
Symfony 3 compliant

### PR DESCRIPTION
**Pull request by meons**

### CODE REVIEW

File: doc/doctrine/uploadable.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ class Document
  public function buildForm(FormBuilderInterface $builder, array $options)
  {
       $builder
-          ->add('file', 'file', array(
+          ->add('file', \Symfony\Component\Form\Extension\Core\Type\FileType::class, array(
```
Comment (maidmaid) [inline doc/doctrine/uploadable.md]: Yep! But, it's better with short notation  `FileType::class` and with `use Symfony\Component\Form\Extension\Core\Type\FileType;` in header, no?

Comment (meons) [reply doc/doctrine/uploadable.md]: Sure! Should I close this PR and let you do the change ?

Comment (maidmaid) [reply doc/doctrine/uploadable.md]: No, you can push a new commit with this change in patch-1 branch :)

**Comment by meons** [comment]
Done ;)

**Comment by maidmaid** [comment]
Thx @meons !

**Pull request merged**